ccd-definition/FixedLists.json

```diff
@@ -513,5 +513,89 @@
     "ID": "PartyType",
     "ListElementCode": "ORGANISATION",
     "ListElement": "Organisation"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "CASE_MANAGEMENT",
+    "ListElement": "Case management",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "FURTHER_CASE_MANAGEMENT",
+    "ListElement": "Further case management",
+    "DisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "ISSUE_RESOLUTION",
+    "ListElement": "Issue resolution",
+    "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "FINAL",
+    "ListElement": "Final",
+    "DisplayOrder": 4
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "OTHER",
+    "ListElement": "Other",
+    "DisplayOrder": 5
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
+    "ListElementCode": "INTERPRETER",
+    "ListElement": "Interpreter",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
+    "ListElementCode": "SPOKEN_OR_WRITTEN_WELSH",
+    "ListElement": "Spoken or written Welsh",
+    "DisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
+    "ListElementCode": "INTERMEDIARY",
+    "ListElement": "Intermediary",
+    "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
+    "ListElementCode": "FACILITIES_OR_ASSISTANCE",
+    "ListElement": "Facilities or assistance for a disability",
+    "DisplayOrder": 4
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
+    "ListElementCode": "SEPARATE_WAITING_OR_SECURITY_MEASURES",
```

Comment:
this code doesn't make sense to me. WAITING ROOM OR SECURITY MEASURES maybe 🤔

Reply:
it was approved by content it is the same as is what is displayed